src/hooks/usePoolData.ts

```diff
@@ -127,6 +129,18 @@ export default function usePoolData(
           account ?? undefined,
         ) as LpTokenUnguarded
       }
+      const keepLPRewardsContract = process.env.production
```

Comment:
nit: i'd add `&& poolName === BTC_POOL_NAME` so we only make this call for that specific pool